project-koku/koku
# adding Azure v2 supported columns

## Jira Ticket

[COST-3805](https://issues.redhat.com/browse/COST-3805)

## Description

This change will add the Azure version 2 cost reports to supported columns for Customer filtered data.

## Testing

1. Checkout Branch
2. Restart Koku
3. Add Azure customer filtered data source
4. Post an Azure v2 generated ingress report
5. Report should be ingested fine.

## Notes

...
